.eslintrc.js

```diff
@@ -417,7 +417,7 @@ module.exports = {
     },
     {
       files: ['e2e/**/*.ts', 'e2e/**/*.js'],
-      rules: { 'import/no-extraneous-dependencies': 'off' },
+      rules: { 'import/no-extraneous-dependencies': 'off', 'no-unused-expressions': 'off' },
```

Comment:
I guess this change is necessary because `.to.be.called` looks like an unused expression otherwise, correct? Well I have to say that's a sad API design decision (it should have been a function), but there's not much we can do but disable the rule indeed.

Reply:
I know, I dislike the API to, it's just prototype pollution. But that's how it's done.